WordPress/Classes/ViewRelated/Reader/ReaderDetailWebviewViewController.swift

```diff
@@ -0,0 +1,83 @@
+import UIKit
+
+protocol ReaderDetailView: class {
```

Comment:
I could be a bit confused by the naming of this protocol thinking it is a view

Reply:
It is intentional. This is generic: it could be a view, a ViewController, a custom class that represents a view, anything!

I'm also not a big fan of declaring protocols like:

```
protocol ReaderDetailViewController

class ReaderDetailViewControllerImpl: UIViewController, ReaderDetailViewController
```

Let me know if this is reasonable for you. Open to suggestions too!

Reply:
I was thinking that the "View" here is confusing. Your decision though

Reply:
We have something similar here: https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/Post/Revisions/ShowRevisionsListManger.swift#L1

Anyway, I can change. I'm just not sure how to call that without reproducing the bad naming convention mentioned above. :(

Reply:
If you think its good and clear then Im ok with it